### PR TITLE
allow references pane to be resized

### PR DESCRIPTION
**Pull request by michaelficarra**

### CODE REVIEW

File: js/menu.js [hunk 1]
```diff
@@ -840,6 +844,29 @@ let referencePane = {
...
+    let startingMousePos = pointerDownEvent.clientY;
+    let startingHeight = parseInt(getComputedStyle(this.$container).height);
+    let moveListener = pointerMoveEvent => {
+      if (pointerMoveEvent.buttons === 0) {
+        removeListeners();
+        return;
+      }
+      let desiredHeight = startingHeight - (pointerMoveEvent.clientY - startingMousePos);
+      this.$container.style.height =
+        Math.max(50, Math.min(visualViewport.height - 150, desiredHeight)) + 'px';
```
Comment (gibson042) [inline js/menu.js:858]: This clamping should probably just be handled by CSS `min-height` and `max-height`.

**Review by gibson042 — APPROVED** [review]
I like this! Can you do the same for the menu pane as well?

File: js/menu.js (hunk 1 above)
Comment (michaelficarra) [reply js/menu.js]: Done.

**Comment by michaelficarra** [comment]
@gibson042 Maybe some other time. Resizing the TOC doesn't seem all that useful to me.

**Comment by gibson042** [comment]
> @gibson042 Maybe some other time. Resizing the TOC doesn't seem all that useful to me.

https://tc39.es/proposal-temporal/#sec-properties-of-the-temporal-plaindatetime-prototype-object
screenshot of Temporal menu pane showing ellipses hiding Temporal.PlainDateTime.prototype property names

**Comment by michaelficarra** [comment]
@gibson042 Yeah but there might be other solutions, like allowing horizontal scroll in the TOC

image

**Comment by gibson042** [comment]
I'd want the pane to be resizable with or without horizontal scrolling.

**Comment by michaelficarra** [comment]
@gibson042 That's fair. You should open an issue to track it.

**Comment by gibson042** [comment]
Or you can add it to this PR, which doesn't have an associated issue AFAICT.

**Comment by michaelficarra** [comment]
@gibson042 I would prefer to have this PR only address the references pane. I ask that you open an issue to track the TOC usability so that, when this PR is merged, we do not forget about the issue you raised.

**Comment by gibson042** [comment]
#549

**Review by bakkot — APPROVED** [review]

**Pull request merged**